bitnami/argo-cd/Chart.yaml

```diff
@@ -30,4 +30,4 @@ sources:
   - https://github.com/argoproj/argo-cd/
   - https://github.com/bitnami/bitnami-docker-dex
   - https://github.com/dexidp/dex
-version: 3.1.10
+version: 3.2.0
```

Comment:
I think the new version should only bump the `patch`, as this does not include a new feature but a bugfix.